src/store/cartSlice.ts

```diff
@@ -133,6 +133,7 @@ export const createCartSlice: StateCreator<
       const reqItems: ICartPushItems = { items: itemsIds }
 
       await createCart(token, reqItems)
+      await fetchCart()
```

Comment:
I don't understand the purpose of this line. `fetchCart` sends `GET /api/v1/cart`, but the returned data isn't used anywhere (correct me, if I'm wrong). So what is the aim of this extra(?) request to the server?

Reply:
Yes! I apologize, I forgot to delete the call of this request. I checked what the GET request with goods returns from the server and compared it with the state when I dealt with the logic of the cart and its goods! I delete it.